### DISCUSSION TITLE
suggestion

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/Nestly-RealState-System#17] [Ideas]
Waxaan idinkula talinaa in folder ka JS aad kaliya hal folder kuso reebtaan kadibna dhamaan files-ka `JS` ku shubtaan. sido kalena css-ta, file ka 1aad ee page kana inaa ka dhigtaan `index` instead of `home`, last but not least, header iyo footer ka dhiga global. sida inaa u sameysaan file css iyo js u gooni ah kana wacdaan file walba aad oga baahataan.

**Comment by sharafdin** [comment]
sido kalena yaan ilaabin in folder ka html aad ku bedeshaan pages.

**Reply by A-M-Ahmed** [reply]
Bro haa I got it. Thank you man

**Reply by A-M-Ahmed** [reply]
> sido kalena yaan ilaabin in folder ka html aad ku bedeshaan pages.

Bro waan sameyay waa fir kartaa waana mahad snthy bro

**Reply by sharafdin** [reply]
good lkn waxad ilowdeen in `folder` ka `html` aad u bixisaan `pages`, sido kalena `assets` waad qaladeen oo `assert` ayaad dhahdeen.

**Reply by A-M-Ahmed** [reply]
> good lkn waxad ilowdeen in `folder` ka `html` aad u bixisaan `pages`, sido kalena `assets` waad qaladeen oo `assert` ayaad dhahdeen.

ok bro hada habenaa

**Reply by A-M-Ahmed** [reply]
> good lkn waxad ilowdeen in `folder` ka `html` aad u bixisaan `pages`, sido kalena `assets` waad qaladeen oo `assert` ayaad dhahdeen.

Hada fiiri bro ok miyaa

**Reply by sharafdin** [reply]
good waye hadeer lkn its better in aad files ka small ka dhigtaan. assets instead of Assets. iyo pages instead of Pages.

**Reply by A-M-Ahmed** [reply]
ok
